docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -89,9 +89,9 @@ The ``list_outer_rows_mosaic.html.twig`` is the name of one mosaic's tile. You s
 
     {% block sonata_mosaic_description %}
         {% if admin.hasAccess('edit', object) and admin.hasRoute('edit') %}
-            <a href="{{ admin.generateUrl('edit', {'id' : object|sonata_urlsafeid(admin) }) }}">{{ meta.title|u.truncate(40) }}</a>
+            <a href="{{ admin.generateObjectUrl('edit', object) }}">{{ meta.title|u.truncate(40) }}</a>
         {% elseif admin.hasAccess('show', object) and admin.hasRoute('show') %}
-            <a href="{{ admin.generateUrl('show', {'id' : object|sonata_urlsafeid(admin) }) }}">{{ meta.title|u.truncate(40) }}</a>
```

Comment:
If we cannot deprecate `sonata_urlsafeid` (https://github.com/sonata-project/SonataAdminBundle/issues/6963#issuecomment-808874731), 
Can we at least deprecate the param admin of the `sonata_urlsafeid` filter ?